Dockerfile

```diff
@@ -1,6 +1,9 @@
-FROM debian:bullseye-slim
+FROM gcr.io/distroless/cc
```

Comment:
:heart_eyes: 

How small is the image after this?

Reply:
![image](https://user-images.githubusercontent.com/224613/143519468-a83b93ef-16e7-4bbb-a546-9b7c5beea621.png)

The one on the top is the new one :)

Reply:
Nice! Still bigger than I expected but that is pretty good :)